build/config/ozone_extra.gni

```diff
@@ -5,18 +5,18 @@
 # This list contains the name of external platforms that are passed to the
 # --ozone-platform command line argument or used for the ozone_platform build
 # config. For example ozone_external_platforms = [ "foo1", "foo2", ... ]
-ozone_external_platforms = []
```

Comment:
add the values with a `use_starboard` check?

Reply:
It looks like we don't currently have [use_starboard defined](https://github.com/search?q=repo%3Ayoutube%2Fcobalt%20use_starboard&type=code).

Would it mechanically differ from is_cobalt or use_starboard_media? If so I can add it, but it seems like [is_cobalt](https://github.com/search?q=repo%3Ayoutube%2Fcobalt%20is_cobalt&type=code) might match what we want, set to false for android builds and linux-x86x11 (but not for linux-cobalt-x86x11).

Reply:
You can use `is_starboard`, defined (transitively) in `build/config/BUILDCONFIG.gn`. Using `is_cobalt` here would break the layer dependencies IMHO.

Reply:
Done.